transport/client.go

```diff
@@ -311,7 +315,7 @@ func Dial(address string, tr *Transport) (*tls.Conn, error) {
 
 	for _, chain := range state.VerifiedChains {
 		for _, cert := range chain {
-			revoked, ok := revoke.VerifyCertificate(cert)
```

Comment:
As I understood, this example can handle only one var per app. What if app serves several TLS enabled listeners with different certs and CRL sets?